commit b8cc937d7aed6900678ec2f024e96635b01a729b (refs/changes/67/1767/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-04-11T21:05:01+00:00 (2 years, 6 months ago)
    
    MANTA-3217 muskie still not using pinger for authcache
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/etc/config.coal.json b/etc/config.coal.json
index 76947d7..bbe0741 100644
--- a/etc/config.coal.json
+++ b/etc/config.coal.json
@@ -43,6 +43,7 @@
         "initialDomains": [
           "authservice.coal.joyent.us"
         ],
+        "ping": "/ping",
         "pingInterval": 60000,
         "tcpKeepAliveInitialDelay": 10000,
         "spares": 8,
@@ -92,6 +93,7 @@
         "delay": 250,
         "maxIdleTime": 10000,
         "maxClients": 50,
+        "ping": "/ping",
         "pingInterval": 14400000,
         "retry": {
           "retries": 2
diff --git a/main.js b/main.js
index baef401..e23fb92 100644
--- a/main.js
+++ b/main.js
@@ -222,7 +222,7 @@ function createCueballHttpAgent(cfg) {
          * Note that this path doesn't actually have to be handled by the
          * authcache (any non-5xx response code is accepted, e.g. 404 is fine).
          */
-        ping: '/ping',
+        ping: sharkCfg.ping,
         pingInterval: sharkCfg.pingInterval,
         tcpKeepAliveInitialDelay: sharkCfg.maxIdleTime,
 
diff --git a/sapi_manifests/muskie/template b/sapi_manifests/muskie/template
index f40f604..5d6bb6a 100644
--- a/sapi_manifests/muskie/template
+++ b/sapi_manifests/muskie/template
@@ -59,6 +59,7 @@
       timeout on the authcache side from closing our sockets (and potentially
       racing with our use of that socket).
     }}
+    "ping": "/ping",
     "pingInterval": 60000,
 
     {{!
@@ -134,6 +135,7 @@
     "maxClients": 50,
 
     {{! nginx has much longer max idle times than node.js }}
+    "ping": "/ping",
     "pingInterval": 14400000,
 
     {{!
